static/app/utils/useHoverOverlay.tsx

```diff
@@ -266,14 +266,35 @@ function useHoverOverlay(
    */
   const wrapTrigger = useCallback(
     (triggerChildren: React.ReactNode) => {
-      const props = {
+      const makeProps = (
+        componentProps: Partial<
+          Pick<
+            React.HTMLAttributes<any>,
+            'onFocus' | 'onBlur' | 'onPointerEnter' | 'onPointerLeave'
+          >
+        >
+      ) => ({
+        // !!These props are always overriden!!
         'aria-describedby': describeById,
         ref: setTriggerElement,
```

Comment:
This is unfortunate, but we need react 19 to be able to fix it. If I attempt to compose the ref it react to error as the ref is a "reserved prop"